sqliter-driver/src/nativeCommonMain/kotlin/co/touchlab/sqliter/interop/Types.kt

```diff
@@ -3,5 +3,5 @@ package co.touchlab.sqliter.interop
 import cnames.structs.sqlite3_stmt
 import kotlinx.cinterop.CPointer
 
-typealias SqliteDatabasePointer = CPointer<co.touchlab.sqliter.sqlite3.sqlite3>
+typealias SqliteDatabasePointer = CPointer<cnames.structs.sqlite3>
```

Comment:
this was a surprising change to make, it seems something in 1.9.20 changed cinterop packaging?

Reply:
Probably a regression of https://youtrack.jetbrains.com/issue/KT-49034

You can work around it by defining the struct manually in your .def like this: https://github.com/cashapp/zipline/blob/ce04291109a5f2e4487bc762bc14a681013836c9/zipline/src/nativeInterop/cinterop/quickjs.def#L7-L10

Reply:
I'm probably holding it wrong, but I tried a few variations e.g. `typedef struct sqlite3 {} sqlite3;` , `typedef struct sqlite3;` etc and all seemed to have the same issue.

Reply:
I'll try on Zipline on Tuesday and see what happens.